streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -679,10 +689,13 @@ private void initializeTaskTime(final Map<TopicPartition, OffsetAndMetadata> off
         return purgableConsumedOffsets;
     }
 
-    private void initializeTopology() {
+    private void initializeTopology(final boolean updatePartitionQueues) {
```

Comment:
Let's move the updatePartitions logic out of `initializeTopology`, instead just call it inside resume

Reply:
moved